docs/layouts/shortcodes/generated/kubernetes_operator_config_configuration.html

```diff
@@ -74,6 +74,12 @@
             <td>Duration</td>
             <td>The interval before a savepoint trigger attempt is marked as unsuccessful.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.periodic.savepoint.interval</h5></td>
```

Comment:
Do we expect this property being set globally?

Reply:
Users can choose to set it globally or override on a per job level, it works both ways.

Reply:
I can change this so setting it to 0 would turn it off explicitly